src/char/inter.c

```diff
@@ -148,233 +149,18 @@ static void inter_do_final_msg(void)
 	for (i = 0; i < MAX_JOB_NAMES; i++)
 		aFree(msg_table[i]);
 }
-/* from pc.c due to @accinfo. any ideas to replace this crap are more than welcome. */
```

Comment:
I removed this comment since I _think_ the main issue here was the manual duplication. If you guys think we should keep the comment for further improvement, just let me know (or if there is a suggestion to improve it already , I can do it)